videoprocess/vpphdr_tm.cpp

```diff
@@ -557,6 +557,11 @@ bool read_frame_to_surface(FILE *fp, VASurfaceID surface_id)
         u_size = (va_image.width / 2 * bytes_per_pixel) * (va_image.height >> 1);
 
         src_buffer = (unsigned char*)malloc(frame_size);
+        if (src_buffer)
+        {
+            printf("malloc buffer failed \n");
+            return false;
+        }
```

Comment:
If malloc failed, maybe we also need to do some clean job(vaUnmapBuffer and vaDestroyImage)

Reply:
ooh! yes, Thanks for reminding.